src/pages/office/cat-cpc/office.cat-cpc.page.ts

```diff
@@ -341,6 +364,16 @@ export class OfficeCatCPCPage extends BasePageComponent {
     this.store$.dispatch(new AdditionalInformationChanged(additionalInformation));
   }
 
+  activityCodeChanged(activityCodeModel: ActivityCodeModel) {
+    const showMeQuestion = this.form.controls['showMeQuestion'];
+    if (showMeQuestion) {
+      if (showMeQuestion.value && showMeQuestion.value.code === 'N/A') {
```

Comment:
could use the `get` function from `lodash` library to avoid the complexity of these if statements

Reply:
addressed, testing it manually showed no obvious change in behaviour and unit tests pass as normal regarding it so can assume working as expected

Reply:
Yep, looks good!